x/msgfees/keeper/params.go

```diff
@@ -9,7 +9,7 @@ import (
 // GetParams returns the total set of distribution parameters.
 func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 	return types.Params{
-		FloorGasPrice: k.GetFloorGasPrice(ctx),
+		FloorGasPrice: uint32(int32(k.GetFloorGasPrice(ctx).Amount.Uint64())),
```

Comment:
See also here ... if that Param was a coin then this gets much easier.

Reply:
hmmm..i was trying not to include the default denom here (since it is specified via another config) but i'll change the param to sdk.Coin now

Reply:
Param is of `sdk.Coin` type now